src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -1980,6 +1980,11 @@ public static int getConcurrentValidations()
         return conf.concurrent_validations;
     }
 
+    public static int getConcurrentIndexBuilds()
```

Comment:
the method should be probably called "getConcurrentIndexBuilders" to be aligned with the terminology we use for other methods around here.

Reply:
Good catch on both; revised.